integration-tests/smartAuth.test.ts

```diff
@@ -0,0 +1,66 @@
+import { AxiosInstance } from 'axios';
+import { getFhirClient } from './utils';
+
+/*
+ *  DDB Set Up on SMART Integration Env
+ *  Sherlock Holmes (Patient)
+ *     id: 92e0d921-bb19-4cae-a3cc-9d3c5bcf7a39
+ *     reference: Patient Mycroft Holmes
+ *     reference: Practitioner Joseph Bell
+ *  Mycroft Holmes (Patient)
+ *     id: cf52937f-a28f-437e-bfac-2228f5db6801
+ *     reference: Patient Sherlock Holmes
+ *     reference: Practitioner Joseph Bell
+ *  John Watson (Patient)
+ *     id: 7965ea12-7ecd-46cd-9ec1-340400c9548c
+ *  Joseph Bell
+ *     id: 7cbe5ea4-826d-4de6-86d9-18644b1cc5b7
+ */
+
+const sherlockId = '92e0d921-bb19-4cae-a3cc-9d3c5bcf7a39';
+const mycroftId = 'cf52937f-a28f-437e-bfac-2228f5db6801';
+
+async function getPatient(client: AxiosInstance, patientId: string | undefined = undefined) {
+    const url = patientId ? `/Patient/${patientId}` : '/Patient';
+
+    return client.get(url);
+}
+
+test('Successfully read Sherlock and Mycroft records', async () => {
```

Comment:
I think that for the test names it's better to avoid names like Sherlock and use their "roles" instead. i.e. "Successfully read own user record and records that reference the user"

Reply:
updated